Lib/test/test_multiprocessing_with_big_output.py

```diff
@@ -0,0 +1,25 @@
+import unittest
+from multiprocessing.pool import Pool
+from time import sleep
+
+
+def func():
+    return 'test' * (10 ** 9)
```

Comment:
Please use a binary string (`b"test"`), and add a comment stating the expected memory consumption.